app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
```

Comment:
Place each argument in the `__init__` on the new line to improve the readability.